cmd/installer/pack.go

```diff
@@ -460,7 +460,20 @@ func (p *PackType) extractEula(packPath string) error {
 
 	eulaFileName := packPath + "." + path.Base(p.Pdsc.License)
 
-	log.Infof("Extracting embedded license to %v", eulaFileName)
+	if utils.GetEncodedProgress() {
+		log.Infof("[L:F\"%s\"]", eulaFileName)
+	} else {
+		log.Infof("Extracting embedded license to %v", eulaFileName)
+	}
+
+	if utils.FileExists(eulaFileName) {
```

Comment:
Why do we have check 2 times the existence of the same file? Are we expecting that the first check won't delete the file?

Reply:
Yes, instead of having a complicated error handling (file just needs to be removed) I added the Exists check twice.